tine20/Timetracker/Export/definitions/ts_default_ods.xml

```diff
@@ -7,51 +7,144 @@
+ </column>
+  <column>
+    <identifier>start_date</identifier>
+	<header>start_date</header>
+    <Type>date</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>start_time</identifier>
+	<header>start_time</header>
+    <Type>time</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>duration</identifier>
+	<header>duration</header>
+    <Type>int(11)</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>description</identifier>
+	<header>description</header>
+    <Type>longtext</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>created_by</identifier>
+	<header>created_by</header>
+    <Type>varchar(40)</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>creation_time</identifier>
+    <header>creation_time</header>
+	<Type>datetime</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>last_modified_by</identifier>
+	<header>last_modified_by</header>
+    <Type>varchar(40)</Type>
+	<width>2,5cm</width>
+   </column>
+  <column>
+    <identifier>last_modified_time</identifier>
+	<header>last_modified_time</header>
+    <Type>datetime</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>is_deleted</identifier>
+	<header>is_deleted</header>
+    <Type>int(11)</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>deleted_by</identifier>
+	<header>deleted_by</header>
+    <Type>varchar(40)</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>deleted_time</identifier>
+	<header>deleted_time</header>
+    <Type>datetime</Type>
+	<width>2,5cm</width>
+ </column>
+  <column>
+    <identifier>seq</identifier>
+	<header>seq</header>
+    <Type>int(11)</Type>
+	<width>2,5cm</width>
+  </column>
         <column>
-            <identifier>account_id</identifier>
-            <header>Staff Member</header>
-            <type>account_id</type>
-            <field>accountDisplayName</field>
-        </column>
-        <column>
-            <identifier>duration</identifier>
-            <header>Duration</header>
-            <type>float</type>
-            <divisor>60</divisor>
-            <number>1</number>
-        </column>
-        <column>
-            <identifier>is_billable_combined</identifier>
-            <header>Billable</header>
-            <type>float</type>
-        </column>
-        <column>
-            <identifier>is_cleared_combined</identifier>
-            <header>Cleared</header>
-            <type>float</type>
+            <identifier>Einsatzort</identifier>
+            <header>Einsatzort</header>
+            <type>string</type>
+            <width>2,5cm</width>
         </column>
-    </columns>
+  <column>
+    <identifier>invoice_id</identifier>
+	<header>invoice_id</header>
+    <Type>varchar(255)</Type>
+	<width>2,5cm</width>
+  </column>
+  <column>
+    <identifier>end_time</identifier>
+	<header>end_time</header>
+    <Type>time</Type>
+	<width>2,5cm</width>
```

Comment:
didn't check it, but quite sure we wouldn't implement cm with a , as a decimal separator. were did you get this piece from? is it actually working?

Reply:
It is working on 
Tine 2018.08.4 on Ubuntu 18.04 (repo: https://packages.tine20.org/ubuntu/ bionic)
the "type" is a copy'n paste from MySqlWorkbenchexport. excuse me I didn't take care of  casesensitives. Ijust tetsted it in my installation and it worked so far.